packages/core/src/logging/Logger.ts

```diff
@@ -50,3 +51,5 @@ export interface LoggerOptions {
   highlighter?: Highlighter;
   usesReplicas?: boolean;
 }
+
+export type LoggerContext = Pick<LogContext, 'label'>;
```

Comment:
maybe better this way so its universal way to pass params down to the logger?

```suggestion
export type LoggerContext = { label?: string } & Dictionary;
```

and add a note about that to the section about custom loggers

Reply:
This change was also needed for `LogContext`... we might consider refactoring these at some point... one acts as a sort of public type where the other is more focused on internals unless someone uses `LoggerFactory`.

Reply:
Also I personally prefer picking the `label` just to keep as much overlap as possible. If you still want that to be unique on it's own, let me know and I can fix it